code/_helpers/icons.dm

```diff
@@ -631,7 +631,6 @@ proc/ColorTone(rgb, tone)
 /*
 Get flat icon by DarkCampainger. As it says on the tin, will return an icon with all the overlays
 as a single icon. Useful for when you want to manipulate an icon via the above as overlays are not normally included.
-The _flatIcons list is a cache for generated icon files.

```

Comment:
the flaticons list mentioned here does not exist. so i removed part of this comment